# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/patches/pkg_device-mapper.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2006-06-21 16:50:57.000000000 +0200
+++ ./configure.in	2006-06-21 16:51:53.000000000 +0200
@@ -86,7 +86,6 @@
 dnl -- Checks for functions
 AC_CHECK_FUNCS(memset mkdir rmdir strchr strdup strerror strrchr uname,,AC_MSG_ERROR(bailing out))
 AC_FUNC_CLOSEDIR_VOID
-AC_FUNC_MALLOC
 AC_FUNC_STAT
 AC_FUNC_VPRINTF
 
